eo-maven-plugin/src/main/java/org/eolang/maven/PullMojo.java

```diff
@@ -150,4 +157,57 @@ private Path pull(final String name) throws IOException {
         return src;
     }
 
+    /**
+     * Fallback which can swap primary/secondary repos.
+     * @since 1.0
+     */
+    public static final class FallbackSwapOy implements Objectionary {
```

Comment:
@mximp I think it would be better to make it an outside class, like all others that start with `Oy`

Reply:
@yegor256 The reason I made it nested is that it's quite specific implementation of Oy tightly bound to `PullMojo`. For now there is only one use case for this - Oy build strategy for `PullMojo`.
Wrapping this logic into class simplifies testing of it and usage, but I don't see a reason to make it standalone.

Reply:
@mximp all other `Oy` classes are only used by `PullMojo`. This one is no different.